app/models/task.rb

```diff
@@ -14,16 +14,23 @@
 # # - Started at
 # # - Completed at
 
-# class Task < ApplicationRecord    
-#     validates :code, presence: true
-#     validates :customer_id, presence: true    
-#     validates :requested_by, presence: true   
-#     validates :subject, presence: true 
-#     validates :description, presence: true 
-#     validates :type, presence: true 
-#     validates :status, presence: true 
-#     validates :responsible, presence: true 
-#     validates :assignee, presence: true 
-#     validates :started_at, presence: true     
-#     validates :completed_at, presence: true                                 
-# end
\ No newline at end of file
+ class Task < ApplicationRecord    
+    belongs_to :customer, class_name:"Company" 
```

Comment:
Nitpick, create a space between `class_name:`and `"Company"`

Reply:
Done